core/src/test/java/org/elasticsearch/index/query/MatchNoneQueryBuilderTest.java

```diff
@@ -0,0 +1,24 @@
+package org.elasticsearch.index.query;
+
+import org.apache.lucene.search.Query;
+import org.elasticsearch.common.lucene.search.Queries;
+
+import java.io.IOException;
+
+public class MatchNoneQueryBuilderTest extends BaseQueryTestCase {
```

Comment:
given that we prefer to keep the match_none internal and we are removing its parser, we can drop this test too.